test/Graphics/Image.test.cpp

```diff
@@ -190,44 +190,32 @@ TEST_CASE("[Graphics] sf::Image")
     {
         SECTION("Invalid size")
         {
-            CHECK(!sf::Image({10, 0}, sf::Color::Magenta).saveToFile("test.jpg"));
-            CHECK(!sf::Image({0, 10}, sf::Color::Magenta).saveToFile("test.jpg"));
+            CHECK(!sf::Image({10, 0}, sf::Color::Magenta).saveToFile("test.jpg", sf::Image::SaveFormat::JPG));
+            CHECK(!sf::Image({0, 10}, sf::Color::Magenta).saveToFile("test.jpg", sf::Image::SaveFormat::JPG));
         }
 
         const sf::Image image({256, 256}, sf::Color::Magenta);
 
-        SECTION("No extension")
-        {
-            CHECK(!image.saveToFile("wheresmyextension"));
-            CHECK(!image.saveToFile("pls/add/extension"));
-        }
```

Comment:
No extension is still a valid test case, but now it should pass, given the passed SaveFormat